frontend/service/agents/PredictTrader.ts

```diff
@@ -38,8 +42,11 @@ export abstract class PredictTraderService extends StakedAgentService {
 
     const provider = PROVIDERS[chainId].multicallProvider;
 
+    const mechContract =
+      MECHS[chainId][stakingProgramConfig.mechType!].contract;
+
     const contractCalls = [
-      // mechContract.getRequestsCount(agentMultisigAddress),
+      mechContract.getRequestsCount(agentMultisigAddress),
```

Comment:
idk where it's used, but we commented the call and forgot to comment the first item in the result, so everything was shifted 😄